source/reviews/comments.txt

```diff
@@ -0,0 +1,112 @@
+  to be corrected. Issues are **always** blocking and no PR should be merged with
+  an open issue.
+- Question: You (the reviewer) have a question about something in a PR.
+  **Do not use** questions as in a Socratic style. If there is a logical, factual, or
+  grammatical error use the proper comment label instead (issue:). Questions must
+  be answered but are **never** blocking, i.e. they do not require a content revision.
+- Comment: You (the reviewer) want to say something about a particular piece
+  of content. Do not use comments as a form of "passive issue". Comments **never** block
+  PR approval and do not require addressing by the author.
+- Suggestion: A way to propose a change as part of a PR comment. It is not required
+  to offer suggestions.
+
+
+.. example::
+
+   .. code:: sh
+
+      issue:
+      suggestion:
+      question:
+      comment:
+
+Courtesy
+++++++++
+
+In general, it is important to be :ref:`courteous and respectful <pr-courtesy>`
+while also being clear and helpful to the author whose content you are
+reviewing. One way to do this is to be sure that you are always making comments
+about the *content* and never making comments about the *author*. You don't
+always have to follow this practice, but you should definitely use it when
+saying something that might otherwise be upsetting or contentious.
+
+.. example::
+
+   Bad: \*\*question\*\* Why did **you** use a semicolon to join two unrelated
+   clauses?
+
+   Good: \*\*suggestion: (blocking)\*\* Semicolons should only be used to
+   join two related independent clauses in place of a comma and coordinating
+   conjunction. Here's how you'd use a semicolon in this instance...
+
+
+Explain Why
++++++++++++
+
+One thing you'll notice about the "good" example from above is that it helps the
+author understand *why* you are making your comment. You don't always need to
+include this information in your review comments, but sometimes it's appropriate
+to give a bit more explanation around your intent, the best practice you're
+following, or how your suggestion improves code health.
+
+Giving Guidance
++++++++++++++++
+
+**In general it is the author's responsibility to fix a PR, not the reviewer's.**
+You are not required to do detailed design of a solution or write
+content for the author.
+
+This doesn't mean the reviewer should be unhelpful, though. In general you
+should strike an appropriate balance between pointing out problems and providing
+direct guidance. Pointing out problems and letting the developer make a decision
+often helps the author learn, and makes it easier to do reviews. It also
+can result in a better solution, because the author is closer to the content
+than the reviewer is.
+
+That said, sometimes direct instructions, suggestions, or even content are more
+helpful. The primary goal of review is to get the best PR possible. A
+secondary goal is improving the skills of authors so that they require less
+and less review over time.
```

Comment:
Suggestion: This value judgment seems central to how we conduct reviews. Should this be stated clearly outside of this section in a prominent location?